lms/models/application_instance.py

```diff
@@ -142,11 +142,7 @@ def update_lms_data(self, params: dict):
         ):
             # If we already have a LMS guid linked to the AI
             # and we found a different one report it to sentry
-            LOG.exception(
-                "Application Instance ID:%s launched in a different LMS install",
-                self.id,
-            )
-            return
+            raise ValueError("Application Instance launched in a different LMS install")
```

Comment:
I think this does deserve a custom exception as we're catching it with an exception view. There are just so many ways a `ValueError` could be raised that aren't this and would trigger the dialog.

Reply:
Absolutely :+1: